packages/create-app/template-vanilla-ts/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+      "target": "ESNext",
+      "module": "ESNext",
+      "moduleResolution": "node",
```

Comment:
Think we should add `"lib": ["esnext", "dom"],`

Reply:
I'm fine with that!

To the best of my knowledge (just opened [a PR on the subject](https://github.com/microsoft/TypeScript-Website/pull/1736), so should have confirmation soon): Dropping `"lib"` results in a default value being used, controlled by `"target"` -- in this case, since `"target"` is `"ESNext"`, `"lib"` will be set to `"ESNext.Full"` -- which cannot be set directly.

[`"ESNext.Full"`](https://github.com/microsoft/TypeScript/blob/master/lib/lib.esnext.full.d.ts) is (currently, it's a moving target) the equivalent of:
`["esnext", "dom", "webworker.importscripts", "scripthost", "dom.iterable"]`

Reply:
Yeah, I think specifying `lib` could be more explicit. Where we also have it for Vue TS

https://github.com/vitejs/vite/blob/de4e08d10e92d3a2411b0393dba6771293fcebbc/packages/create-app/template-vue-ts/tsconfig.json#L11

Reply:
@antfu, yup, all of the template `tsconfig.json` have `"lib"`, except for for the [`template-svelte-ts` `tsconfig.json`](https://github.com/vitejs/vite/blob/main/packages/create-app/template-svelte-ts/tsconfig.json), been considering floating the idea of making them a little more consistent. Obviously some of the differences are necessary, but others seem arbitrary.